CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.4.0 - UNRELEASED
+### Features
+* Dnsmasq host alias (`servicehost`) on all workers and controllers (#25, #46)
+* Nginx service router address, port and domain now configurable (#44)
+* Docker 1.10.0 (#47)
+* Mesos 0.27.0 (#35)
+* Marathon 0.15.1 (#51)
+* Marathon syslog output is now suppressed (#54)
+* README improvements (#48)
+* Setting up a cluster now requires that only the controllers are able to resolve the hostnames of the workers (no other lookups are required) (#35)
```

Comment:
The current phrasing of this implies that non-controllers being able to resolve worker hostnames might break setup. Maybe reword it slightly?